code/datums/diseases/advance/advance.dm

```diff
@@ -254,8 +254,8 @@
 			SetSpread(DISEASE_SPREAD_BLOOD)
 
 		permeability_mod = max(CEILING(0.4 * properties["transmittable"], 1), 1)
-		cure_chance = 15 - clamp(properties["resistance"], -5, 5) // can be between 10 and 20
-		stage_prob = max(properties["stage_rate"], 2)
+		cure_chance = clamp(7.5 - (0.5 * properties["resistance"]), 5, 10) // can be between 5 and 10
+		stage_prob = max(0.5 * properties["stage_rate"], 1)
```

Comment:
What the hell is going on here?

Reply:
Making these probabilities also conform with the proper rates.